inst/shiny-examples/20_gs-explorer/server.R

```diff
@@ -38,9 +38,9 @@ shinyServer(function(input, output, session) {
   output$logoutButton <- renderUI({
     if(!is.null(access_token())) {
       # Revoke the token too? use access_token$revoke()
-      actionButton("logoutButton",
-                   label = a("Logout",
-                             href = getOption("googlesheets.shiny.redirect_uri")))
+      tags$a("Logout",
+            href = getOption("googlesheets.shiny.redirect_uri"),
```

Comment:
@jennybc should this be `googlesheets.webapp.redirect_uri` instead of `googlesheets.shiny.redirect_uri`? I'm just asking because this code didn't seem to work for me, and it doesn't look like that option exists much elsewhere in the codebase